docs/content/AvatarStack.mdx

```diff
@@ -2,6 +2,9 @@
 title: AvatarStack
 ---
 
+import {Props} from '../src/props'
+import {AvatarStack} from '@primer/components'
+
 AvatarStack is used to display more than one Avatar in an inline stack.
```

Comment:
This is not necessary for this PR, but while we're here it might be nice to add some more metadata to the docs:
```suggestion
title: AvatarStack
status: Alpha
source: https://github.com/primer/react/blob/main/src/AvatarStack.tsx
description: Use avatar stack to display multiple avatars in an inline stack.
---

import {Props} from '../src/props'
import {AvatarStack} from '@primer/components'
```

Reply:
😍 I would like that very much. I'm happy to work toward that in a separate stream of work.